cluster-scope/base/core/namespaces/octo-et-security-demo-next/namespace.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/display-name: "OCTO ET Security: Demo for NEXT"
+    openshift.io/requester: octo-et-security
+    op1st/project-owner: octo-et-security
+    op1st/onboarding-issue: "https://github.com/operate-first/support/issues/628"
+    op1st/docs: "https://github.com/orgs/redhat-et/teams/security"
+  name: octo-et-security-demo-next
```

Comment:
This namespace name does not match whats in the support issue (next-security-demo), is that ok?

Reply:
Yes, I think, namespaces should have the group as a prefix, if possible